libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/roadlabel/RoadLabelSurfaceLayer.kt

```diff
@@ -137,5 +106,9 @@ internal class RoadLabelSurfaceLayerDelegate : CarSurfaceLayer() {
             .roundedLabelColor(Color.WHITE)
             .textColor(Color.BLACK)
             .build()
+
+        private val EMPTY_BITMAP = Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888).apply {
```

Comment:
```suggestion
        private val EMPTY_BITMAP = Bitmap.createBitmap(1, 1, Bitmap.Config.ARGB_8888).apply {
```

Size of this bitmap matter?

Reply:
No, this was simply copied from `CarTextLayerHost`.